src/HOHQMesh.jl

```diff
@@ -168,12 +168,19 @@ control file name. For example, `path/to/ControlFile.control` will result in out
 You can activate verbose output from HOHQMesh that prints additional messages and debugging
 mesh information with the keyword argument `verbose`.
 
+To override the maximum number of allowable subdivisions in the quad tree during meshing
+adjust the value of `subdivision_maximum`. The default value of `subdivision_maximum` is 8,
+meaning that elements can be up to a factor of `2^8` smaller than the background grid.
+Note, think before doing this! It could be adjusting the boundary curves, background grid size,
+adding local refinement regions, or some combination may remove the need to adjust
+the subdivision depth.
+
 This function returns the output to `stdout` of the HOHQMesh binary when generating the mesh.
 """
 function generate_mesh(control_file;
                        output_directory="out",
                        mesh_filename=nothing, plot_filename=nothing, stats_filename=nothing,
-                       verbose=false)
+                       verbose=false, subdivision_maximum=8)
```

Comment:
```suggestion
                       verbose=false, subdivision_maximum::Int=8)
```
To ensure halfway reasonable values (even though that still allows negative values, but then again, _some_ level user sanity is mandatory anyways)

Reply:
True, I make a small assumption that users know this input should be a positive integer.

Reply:
Should the safety `::Int` also be added to the `generate_mesh` function in `src/Mesh/Meshing.jl`?

Reply:
Are both functions user facing? If yes, for consistency I'd add it to both.

Reply:
Yeah, they are both user facing. I have added it for consistency.